easybuild/easyconfigs/b/BGEN-enkre/BGEN-enkre-1.1.7-GCC-11.2.0.eb

```diff
@@ -0,0 +1,74 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# we recommend to use --download-timeout=1000 when fetching the files
+
+easyblock = 'CmdCp'
+
+name = 'BGEN-enkre'
+version = '1.1.7'
+
+homepage = 'https://enkre.net/cgi-bin/code/bgen/dir?ci=trunk'
+description = """This repository contains a reference implementation 
+of the BGEN format, written in C++. The library can be used as the 
+basis for BGEN support in other software, or as a reference for 
+developers writing their own implementations of the BGEN format.
+Please cite:
+Band, G. and Marchini, J., "BGEN: a binary file format for imputed genotype and haplotype data", 
+bioArxiv 308296; doi: https://doi.org/10.1101/308296
+"""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://code.enkre.net/bgen/tarball/release/']
+sources = ['%(version)s.tgz']
```

Comment:
either replace with git config checkout or replace release with v%(version)s

Reply:
done

Reply:
No it's not, you haven't fixed the source url at all, it still says "release" and instead you changed the source filename which i explained above makes no difference.

Reply:
Oh, OK.  Replace the word "release" in the URL.  Done.

Reply:
Looks okay to me now.